db/multi_cf_iterator.cc

```diff
@@ -9,35 +9,28 @@
 
 namespace ROCKSDB_NAMESPACE {
 
-void MultiCfIterator::SeekCommon(
-    const std::function<void(Iterator*)>& child_seek_func,
-    Direction direction) {
-  direction_ = direction;
-  Reset();
+template <typename BinaryHeap, typename ChildSeekFuncType>
+void MultiCfIterator::SeekCommon(BinaryHeap& heap,
+                                 const ChildSeekFuncType& child_seek_func) {
```

Comment:
C++ std APIs do not use `const &` to decorate parameterized callable types as function parameter types. See e.g. https://en.cppreference.com/w/cpp/algorithm/all_any_none_of